.gitignore

```diff
@@ -75,3 +75,5 @@ tags
 .vscode/*
 .history
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
+# IDE folders
```

Comment:
`.vscode/*` is also IDE folder but is out of that group... It changes nothing but maybe it makes sense to regroup gitignore

Reply:
It looks like the main block of the gitignore is auto-generated, and I didn't want to mess around with that part :)

I tried using the same site (which has since rebranded to toptal.com) but it spews a ton of lines that seem unnecessary to me.